src/lib/core/theming/_theming.scss

```diff
@@ -82,3 +82,11 @@
     background: $mat-dark-theme-background,
   );
 }
+
+// Approximates an rgba color into a solid hex color, given a background color.
+@function mat-rgba-to-hex($color, $background-color) {
```

Comment:
Is this equivalent to `alpha-blend`?
https://gist.github.com/jelbourn/f890a79a48cb21a5246a

Reply:
It should be pretty close, but I haven't compared the results. We've been using this one for a while now in a couple of places.